server/status.go

```diff
@@ -223,6 +236,72 @@ func (s *statusServer) handleLocalLog(w http.ResponseWriter, r *http.Request, ps
 	w.Write(b)
 }
 
+// parseInt64WithDefault attempts to parse the passed in string. If an empty
+// string is supplied or parsing results in an error the default value is
+// returned.  If an error does occur during parsing, the error is returned as
+// well.
+func parseInt64WithDefault(s string, defaultValue int64) (int64, error) {
+	if len(s) == 0 {
+		return defaultValue, nil
+	}
+	result, err := strconv.ParseInt(s, 10, 0)
+	if err != nil {
+		return defaultValue, err
+	}
+	return result, nil
+}
+
+// handleLocalLog returns the log entires parsed from the log file that occurred
+// after or on the query parameter "starttime" and before or on the query
+// parameter "endtime". Additionally, if a level is specified via the URL, the
+// log entires are scoped to that level. If no "starttime" is provided, then
+// a "starttime" of a day ago is used.  If no "endtime" is provided, the current
+// time is used. The cutoff for entires is defined in log.EntriesCutoff.
+func (s *statusServer) handleLocalLog(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	log.Flush()
+	level, _ := log.SeverityByName(ps.ByName("level"))
```

Comment:
This should return an error on an unrecognized level (just like we return errors for unparseable numbers).